spec/factories/mvi_profiles.rb

```diff
@@ -40,9 +40,9 @@
     given_names { Array.new(2) { Faker::Name.first_name } }
     family_name { Faker::Name.last_name }
     suffix { Faker::Name.suffix }
-    gender { Faker::Medical::Patient.gender }
+    gender { %w[M F].sample }
```

Comment:
Probably getting a bit pendantic, but this could be `Faker::Gender.short_binary_type.capitalize`

Reply:
```
$ rails runner benchmark/test_code.rb
Running via Spring preloader in process 26628
Warming up --------------------------------------
        sample array     1.207M i/100ms
        faker gender     1.000  i/100ms
Calculating -------------------------------------
        sample array     11.539M (± 2.6%) i/s -     57.950M in   5.025488s
        faker gender     51.031k (±13.3%) i/s -    238.663k in   4.927765s
```